tests/unit/style/properties/serialization.rs

```diff
@@ -1114,4 +1114,85 @@ mod shorthand_serialization {
             assert_eq!(s, "none");
         }
     }
+
+    mod animation {
+        pub use super::*;
+        use servo_atoms::Atom;
+        use style::properties::longhands::animation_delay;
+        use style::properties::longhands::animation_direction;
+        use style::properties::longhands::animation_duration;
+        use style::properties::longhands::animation_fill_mode;
+        use style::properties::longhands::animation_iteration_count;
+        use style::properties::longhands::animation_name;
+        use style::properties::longhands::animation_play_state;
+        use style::properties::longhands::animation_timing_function;
+        use style::values::specified::Time as TimeContainer;
+
+        macro_rules! create_animation_properties {
```

Comment:
You probably should be able to specify a vector of each property, since we're testing for cases where the vectors don't match.